chromadb/utils/embedding_functions.py

```diff
@@ -115,7 +129,15 @@ def __init__(
             raise ValueError(
                 "The openai python package is not installed. Please install it with `pip install openai`"
             )
+        try:
+            import tiktoken
 
+            self._tokenize_function = tiktoken.get_encoding("cl100k_base").encode
+        except ImportError:
+            raise ValueError(
+                "The tiktoken python package is not installed. Please install it with `pip install tiktoken`"
+            )
+            self._tokenize_function = None
```

Comment:
This will never be called and this case will never matter since we raise an error. Perhaps we can log instead.

Reply:
I'll just remove the checking and delegate to OAI API.